lib/lev/routine.rb

```diff
@@ -208,9 +208,13 @@ def [](*args, &block)
         result.outputs.send(@express_output)
       end
 
+      def set(options)
```

Comment:
Should the name of this be more explicit, like `set_job_options`?  `set` is maybe a little too generic for me.

Reply:
or if keep as `set` would need to see that `queue` was a key and make this specific call

Reply:
Just wanted to keep the same API as active_job: https://apidock.com/rails/ActiveJob/Core/ClassMethods/set

I could make the keys queue, wait and wait_until specifically, but they would have to be all optional.